analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -158,7 +164,21 @@ class ClearlyDefinedPackageCurationProvider(server: Server = Server.PRODUCTION)
         val (type, provider) = pkgId.toClearlyDefinedTypeAndProvider()
         val curationCall = service.getCuration(type, provider, namespace, pkgId.name, pkgId.version)
 
-        val curation = curationCall.execute().body() ?: return emptyList()
+        val response = try {
+            curationCall.execute()
+        } catch (e: IOException) {
+            e.showStackTrace()
+
+            createAndLogIssue(
```

Comment:
This returns an instance of `OrtIssue` which is not used. Is this intended or should it rather be returned somehow so that it can be added to some OrtResult?

Reply:
Argh, thanks for catching this. Looks like we currently have no way of creating `OrtIssue`s from `CurationProvider`s, so I probably need to rework this quite a bit...

Reply:
As I need this urgently I decided to only log the warning for now.

Reply:
Sure.